lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -1,94 +1,482 @@
-"""
-Helper functions to get the elasticsearch specific configurations
-"""
+from datetime import datetime
+from dateutil import rrule, parser as date_parser
+from dateutil.relativedelta import relativedelta
+from enum import Enum
+from flask import request
+from flask_restful import Resource, abort
+from logging import Logger
+import requests
+from typing import Any, AnyStr, Callable, Dict, Iterable
+from urllib.parse import urljoin
 
-from dateutil import rrule
+from pbench.server import PbenchServerConfig
+from pbench.server.api.auth import Auth
```

Comment:
FYI, I told vscode to sort the imports; the new list is what it chose.

Reply:
Internal? You mean the Pbench packages? Or do we want to separate built-in from external from "internal" with multiple blanks as Peter showed?

I'm not sure how configurable vscode's sorting algorithm is; I'll have to check. Maybe it's just going to prove useless, but I thought I'd try it this time.

Reply:
I think it's worthwhile separating "our" packages from "other people's" packages.  "Our" packages come from our repo, their paths begin with `pbench`, etc.

In an ideal world, I would be OK with separate sections for "built-in" (or "standard") packages versus "third-party" packages, but I'm not completely sure how to tell them apart, and I think it's a difference without a distinction.  So, I vote for just two sections, each in alphabetical order.  (If we want, we can include the leading keyword (`from`/`import`) as part of the alphabetization -- I like the effect of that -- but it's probably better to just alphabetize based on the package name alone, so that it's more obvious when there are multiple imports of the same library using _both_ keywords.)

Reply:
The "tradition" has been to put "import" first and "from x import" after, which is what I've done in the latest commit.

I could definitely get on board with simply alphabetizing by package name regardless of "from", but that's not "how it's been done".

I definitely agree with a substantial lack of desire/motivation to try to figure out for each package whether it's "standard" or "third-party", because that sounds like a potentially massive time sync with no measurable value.

Reply:
I'm OK with the code as it now stands.  (Who gets to resolve this conversation?  Do we have to wait for Peter to push the button?  ;-) )

Reply:
[PEP 8](https://www.python.org/dev/peps/pep-0008/#imports) clearly states, "standard library, related 3rd party, local; with a blank line between".

We have been within each of those groups, we have been separating bare `import` from `from` imports, but that is just to make it easier to sort the bare `import` statements to avoid duplicating them.